### DISCUSSION TITLE
End of Clip Freeze

### DISCUSSION THREAD

**Discussion by Don1776** [smplayer-dev/smplayer#732] [Q&A] [answered]
Normally when playing a clip, at the end the program switches to the logo screen. Is it possible to freeze on the last frame rather than switching to the logo? Thanks! By the way, I love this program.
- Don

**Comment by Raza-0007** [comment, marked as answer]
If your default multimedia engine is mpv, then pass the following option here:

Options>Preferences>Advanced>MPlayer/mpv>Options:>

--keep-open=yes 


Here are the arguments for this command:

--keep-open=no     
[If the current file ends, go to the next file or terminate. (Default.)]

--keep-open=yes      
[Don't terminate if the current file is the only file or the last playlist entry. Equivalent to --keep-open without arguments.]

--keep-open=always     
[Like yes, but also applies to files before the last playlist entry. This means playback will never automatically advance to the next file.]

~Raza

**Reply by Don1776** [reply]
Thanks Raza! That does exactly what I want. We are playing clips from a football game, doing lots of pausing, rewinding and fast forwarding. If we accidentally came to the end of the clip it would terminate and have to be reloaded again. Thanks!
~ Don

**Reply by Raza-0007** [reply]
You are welcome Don!
Take care.

~Raza
